spec/requests/lettings_logs_controller_spec.rb

```diff
@@ -1153,119 +1144,54 @@
-              expect(completed_lettings_log.status).to eq("completed")
               follow_redirect!
-              expect(page).to have_content("This log is from the 2021/2022 collection window, which is now closed.")
+              expect(page).to have_content(/This log is from the \d{4}\/\d{4} collection window, which is now closed\./)
             end
           end
 
-          context "when a lettings log is for a renewal of supported housing, property information does not need to show" do
-            let(:lettings_log) do
-              FactoryBot.create(
-                :lettings_log,
-                owning_organisation: user.organisation,
-                managing_organisation: user.organisation,
-                assigned_to: user,
-                startdate: Time.zone.now,
-                renewal: 1,
-                needstype: 2,
-                rent_type: 3,
-                postcode_known: 0,
-              )
-            end
-
-            before do
-              sign_in user
-            end
+          context "when a lettings log is for a renewal of supported housing" do
+            let(:lettings_log) { create(:lettings_log, :startdate_today, assigned_to: user, renewal: 1, needstype: 2, rent_type: 3, postcode_known: 0) }
 
-            around do |example|
-              FormHandler.instance.use_real_forms!
-              example.run
-              FormHandler.instance.use_fake_forms!
+            it "does not show property information" do
+              get lettings_log_path(lettings_log)
+              expect(page).to have_content "Tenancy information"
+              expect(page).not_to have_content "Property information"
             end
 
             it "does not crash the app if postcode_known is not nil" do
-              expect {
-                get "/lettings-logs/#{lettings_log.id}", headers:, params: {}
-              }.not_to raise_error(ActionView::Template::Error)
-            end
-          end
-
-          context "with a lettings log with a single section complete" do
-            let(:section_completed_lettings_log) do
-              FactoryBot.create(
-                :lettings_log,
-                :conditional_section_complete,
-                assigned_to: user,
-              )
-            end
-
-            before do
-              Timecop.freeze(2021, 4, 1)
-              Singleton.__init__(FormHandler)
-              sign_in user
-              get "/lettings-logs/#{section_completed_lettings_log.id}", headers:, params: {}
-            end
-
-            after do
-              Timecop.unfreeze
-              Singleton.__init__(FormHandler)
-            end
-
-            it "displays a section status for a lettings log" do
-              assert_select ".govuk-tag", text: /Not started/, count: 6
-              assert_select ".govuk-tag", text: /Completed/, count: 1
-              assert_select ".govuk-tag", text: /Cannot start yet/, count: 1
```

Comment:
shout if you think this is testing something important, but it seems to be fairly closely repeating another test above, checking numbers of statuses of sections. again I may be missing context, but this is the only place using the :conditional_section_complete trait that I removed.